services/issue/assignee.go

```diff
@@ -21,9 +21,10 @@ import (
 // DeleteNotPassedAssignee deletes all assignees who aren't passed via the "assignees" array
 func DeleteNotPassedAssignee(issue *models.Issue, doer *user_model.User, assignees []*user_model.User) (err error) {
 	var found bool
+	oriAssignes := make([]*user_model.User, len(issue.Assignees))
+	_ = copy(oriAssignes, issue.Assignees)
```

Comment:
unrelated bugfix/enhancement?

Reply:
Bugfix. Because issue.Assignees will be changed in Togglexxxx method.